tools/tikv-control.md

```diff
@@ -0,0 +1,164 @@
+         write cf value: start_ts: 399650105199951882 commit_ts: 399650105213059076 short_value: "\000\000\000\000\000\000\000\001"
+```
+
+In this command, the key is also the escaped form of raw key.
+
+### Print a specific key value
+
+To print the value of a key, use the `print` command.
+
+### Compact data manually
+
+Use the `compact` command to manually compact TiKV data. If you specify the `--from` and `--to` options, then the flag is also in the form of escaped raw key. You can use the `--db` flag to specify the RocksDB that you need to compact. The optional flag values are `kv` and `raft`.
+
+```bash
+$ tikv-ctl --db /path/to/tikv/db compact -d kv
+success!
+```
+
+### Set a Region to tombstone
+
+The `tombstone` command is usually used in circumstances where the sync-log is not enabled, and some write data of the Raft state machine is lost caused by power down.
+
+In a TiKV instance, you can use this command to set the status of some Regions to Tombstone. Then when you restart the instance, those Regions are skipped. Besides, those Regions need to have enough healthy replicas in other TiKV instances, so as to be able to continue writing and reading through the Raft mechanism.
+
+```bash
+pd-ctl>> operator add remove-peer <region_id> <peer_id>
+$ tikv-ctl --db /path/to/tikv/db tombstone -p 127.0.0.1:2379 -r 2
+success!
+```
+
+> **Note:**
+>
+> - This command only supports the local mode.
+> - The argument of the `--pd/-p` option specifies the PD endpoints without the `http` prefix. Specifying the PD endpoints is to query whether PD can securely switch to Tombstone. Therefore, before setting a PD instance to Tombstone, you need to take off the corresponding Peer of this Region on the machine in `pd-ctl`.
+
+### Force Region to recover the service from multiple replicas failure
+
+Use the `unsafe-recover remove-fail-stores` command to remove the failed machines from the peers list of all Regions. Then after you restart TiKV, these Regions can continue to provide services using the other healthy replicas. This command is usually used in circumstances where multiple TiKV stores are damaged or deleted.
+
+```bash
+$ tikv-ctl --db /path/to/tikv/db unsafe-recover remove-fail-stores 3,4,5
+success!
+```
+
+> **Note:** This command only supports the local mode. It prints `success!` when successfully run.
+
+### Send a `consistency-check` request to TiKV
+
+Use the `consistency-check` command to execute a consistency check among replicas in the corresponding Raft of a specific Region. If the check fails, TiKV itself panics. If the TiKV instance specified by `--host` is not the Region leader, an error is reported.
+
+```bash
+$ tikv-ctl --host 127.0.0.1:21060 consistency-check -r 2
+success!
+$ tikv-ctl --host 127.0.0.1:21061 consistency-check -r 2
+DebugClient::check_region_consistency: RpcFailure(RpcStatus { status: Unknown, details: Some("StringError(\"Leader is on store 1\")") })
+```
+
+> **Note:**
```

Comment:
I think maybe there is something wrong with the format since the color of some words and punctuation marks is red or blue.